README.rst

```diff
@@ -86,7 +123,7 @@ Linux and OSX
 
 .. code-block:: bash
 
-    ./scripts/run_tests.sh
+    .ci/scripts/test_linux.sh
```

Comment:
Do you mean `ci/test/test.sh` ? `test_linux.sh` is using docker I don't think customer has the access to the docker image.